package.json

```diff
@@ -20,19 +20,19 @@
         "_______ Hooks _______": "Yarn hooks.",
         "postinstall": "yarn run patch-package",
         "_______ Library Scripts #####": "Some libraries have their own build scripts.",
-        "build:libs": "lerna run build:lib",
+        "build:libs": "yarn nx run-many --target=build:lib",
```

Comment:
I would probably add `--skip-nx-cache` just for this script for security purposes, because this can affect build output. Other scripts are just validations and can't affect build.

Reply:
that is why I put the `--skip-nx-cache` param to CI codesign builds: https://github.com/trezor/trezor-suite/pull/6816/files#diff-fad8dced0db200747490daa0aa2fb66c1d30378b9cf5ba032ae96e2e5d8825c1R61

but I will put it here to be on the safe side

Reply:
https://github.com/trezor/trezor-suite/pull/6816/commits/a02869f2ab6e215b060c692bf0653ac1db663d0d